index.css

```diff
@@ -11,16 +11,19 @@ main {
   width: 375px;
   height: 5410px;
   background: #fff;
+  margin-left: 30px;
+  display: grid;
```

Comment:
- [ ] Please work on your mobile design to still look as expected. The project is no more responsive at any screen size
### **Take A Look at Your Mobile Design**
![mobile](https://user-images.githubusercontent.com/55596970/167156555-40571d97-88ec-43ca-b6a5-5b347222c867.png)